UnitsNet.Tests/CustomCode/AreaTests.cs

```diff
@@ -118,5 +118,26 @@ public void ToUnit_GivenSIUnitSystem_ReturnsSIQuantity()
             Assert.Equal(0.00129032, inSI.Value);
             Assert.Equal(AreaUnit.SquareMeter, inSI.Unit);
         }
+
+        [Theory]
+        [InlineData(-3.0, -3.0)]
+        [InlineData(-2.0, -0.5)]
+        [InlineData(0.0, 0.0)]
+        [InlineData(3.0, 3.0)]
+        [InlineData(2.0, 0.5)]
+        public void InverseReturnsReciprocalArea(double value, double expected)
+        {
+            var area = new Area(value, AreaUnit.SquareMeter);
+            var inverseArea = area.Inverse();
+
+            AssertEx.Equals(expected, inverseArea.InverseSquareMeters);
```

Comment:
I couldn't for the life of me understand why these tests passed.

Turns out `AssertEx.Equals` does not exist, so this wound up calling `System.Object.Equals()` and this naturally does not throw an assertion exception. 😄

Pushed a fix now.